host/4/bullseye/amd64/python/python311/python311-slim.Dockerfile

```diff
@@ -40,7 +40,7 @@ RUN apt-get update && \
     rm -f /$EXTENSION_BUNDLE_FILENAME_V4 &&\
     find /FuncExtensionBundles/ -type f -exec chmod 644 {} \;
 
-FROM mcr.microsoft.com/mirror/docker/library/python:3.11-slim as python
+FROM mcr.microsoft.com/mirror/docker/library/python:3.11-slim-bullseye as python
```

Comment:
Any specific reason we have `as python` alias here and not in other images?

Reply:
For 3.11 we used different approach where we use golden image, use runtime-deps as the runtime image, install Python dependencies into the python image and copy it over. This would be good for layering and keeping track of items.